experimental/benchmarks/acmeair/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM ubuntu:14.04
+MAINTAINER Gareth Ellis <gareth.ellis@uk.ibm.com>
+RUN apt-get update && apt-get install git openjdk-7-jdk mongodb zip unzip numactl bc wget -y
+RUN mkdir /node 
+RUN date
+WORKDIR /node
+RUN mkdir Jmeter node_undertest node_baseline
+RUN git clone http://github.com/gareth-ellis/benchmarking 
```

Comment:
If you execute the `git clone` and `setupJmeter` in the `RUN` command above alongside `apt-get install`, you can purge `git` `wget` and `unzip` in the same layer keeping the image small. Take a look at the [Node.js Slim Official Image](https://github.com/nodejs/docker-node/blob/master/6.2/slim/Dockerfile#L31) for inspiration.